src/legacy/Test.FAKECore/PackageMgt/FeedSpecs.cs

```diff
@@ -37,7 +37,10 @@ public class when_discovering_the_lastest_FAKE_package
             () => _package.Published.Year.ShouldBeGreaterThanOrEqualTo(2012);
 
         It should_contain_the_license_url = 
-            () => _package.LicenseUrl.ShouldStartWith("https://github.com/fsharp/Fake/blob/master/License.txt") && _package.LicenseUrl.ShouldEndWith("License.txt");
```

Comment:
Sorry, please just merge release/next I already fixed that (and actually made the same mistake :D )

Reply:
Hahaha we would make the worst C# devs 😄